samples: radio_test: Fix sample building with Skyworks FEM (#9726)
nrfconnect/sdk-nrf: doc/nrf/known_issues.rst

NCSDK-13949: TF-M Secure Image copies FICR to RAM on nRF9160
  TF-M Secure Image copies the FICR to RAM address between ``0x2003E000`` and ``0x2003F000`` during boot on nRF9160.

Samples
*******

.. rst-class:: v2-2-0

>   NCSDK-18847: :ref:`radio_test` sample does not build with support for Skyworks front-end module.

Reviewer: ```suggestion
NCSDK-18847: :ref:`radio_test` sample does not build with support for Skyworks front-end module.
  After adding the MSPL FEM Power splitter feature, the sample does not build with support for Skyworks FEM.
```
An issue always needs also the description after the summary.

Author: Ok, I completely changed description.

>   **Workaround:** Do not use the :kconfig:option:`CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC` Kconfig option and replace ``CONFIG_GENERIC_FEM`` with ``CONFIG_MPSL_FEM_SIMPLE_GPIO`` in the sample :file:`CMakeLists.txt` file.

Author: Sure. I was not sure also about this location.

NCSDK-13949: TF-M Secure Image copies FICR to RAM on nRF9160
  TF-M Secure Image copies the FICR to RAM address between ``0x2003E000`` and ``0x2003F000`` during boot on nRF9160.

Samples
*******

.. rst-class:: v2-2-0

NCSDK-18847: :ref:`radio_test` sample does not build with support for Skyworks front-end module.

>   When bulding sample with support for different front-end module than the nRF21540, the sample uses not existing configuration to initialize Tx power data which cause compilation error also source file with code for generic front-end module is not included into build.

Reviewer: ```suggestion
  When building the sample with support for different front-end module than nRF21540, it uses a non-existing configuration to initialize Tx power data.
  This causes a compilation error, because the source file with code for a generic front-end module is not included in the build.
```

Author: Done

NCSDK-13949: TF-M Secure Image copies FICR to RAM on nRF9160
  TF-M Secure Image copies the FICR to RAM address between ``0x2003E000`` and ``0x2003F000`` during boot on nRF9160.

Samples
*******

.. rst-class:: v2-2-0

NCSDK-18847: :ref:`radio_test` sample does not build with support for Skyworks front-end module.

>   When building the sample with support for different front-end module than nRF21540, it uses a non-existing configuration to initialize Tx power data.

Reviewer: Good catch Anna. I was sure there was something I forgot. And yes, it was this different from. But I'd still say **the** sample, as it refers to the Radio test sample mentioned in the line above.
